tests/support/mod.rs

```diff
@@ -362,6 +363,7 @@ fn lines_match(expected: &str, mut actual: &str) -> bool {
 struct ZipAll<T, I1, I2> {
     first: I1,
     second: I2,
+    marker: PhantomData<T>,
```

Comment:
This can actually be removed entirely by placing `Iterator` bounds on `I1` and `I2` and then the type can go away.